tests/snapshots/tests.integration.test_integration_snapshots.test_tracer_trace_across_multiple_forks.snap

```diff
@@ -0,0 +1,36 @@
+[[{"name" "parent"
+   "service" nil
+   "resource" "parent"
+   "error" 0
+   "span_id" 0
+   "trace_id" 0
+   "parent_id" nil
+   "start" 1618357412996234998
+   "duration" 425047115
+   "meta" {"runtime-id" "31d11c17a6c547ccb9169d1347b1379e"}
+   "metrics" {"_dd.agent_psr" 1.0
+              "_sampling_priority_v1" 1
+              "system.pid" 6435
+              "_dd.tracer_kr" 1.0}}
+      {"name" "child1"
+       "service" nil
+       "resource" "child1"
+       "error" 0
+       "span_id" 1
+       "trace_id" 0
+       "parent_id" 0
+       "start" 1618357413012622804
+       "duration" 315314377
+       "metrics" {"_sampling_priority_v1" 1
+                  "_dd.tracer_kr" 1.0}}
```

Comment:
We don't report the system.pid for child spans but I think it'd be valuable here since this span is coming from a new process. Thoughts @DataDog/apm-python?

Reply:
👍🏻 would be good to have the key here, but we can always ignore it when doing the comparison

Reply:
Oh to be clear I don't mean the snapshot. We don't report include the pid on subprocess child spans (only do it on parent spans iirc)